nodebuilder/util/env.go

```diff
@@ -0,0 +1,17 @@
+package util
```

Comment:
Please don't. The previous version where you put it into p2p was ok. The notion of Bootstapper is logically a p2p thing, so it should be there rather than some utility. Ther is bootstrapp.go file where you can put this to.